src/main/java/com/soen343/gms/model/Job.java

```diff
@@ -0,0 +1,47 @@
+package com.soen343.gms.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table
+@Getter
+@Setter
+public class Job {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String description;
+    private String mechanic;
+
+    @Enumerated(EnumType.STRING)
+    private JobState state;
+
+    @OneToMany(cascade=CascadeType.ALL)
+    private List<Task> tasks = new ArrayList<Task>();
+
+    @Enumerated(EnumType.STRING)
+    public JobState getState() {
+        return this.state;
+    }
+    public String toString() {
+
+        String taskToString ="";
+        for(int i = 0; i < tasks.size(); i++)
+        {
+            taskToString += tasks.get(i).toString() + "\n";
+        }
+        return "Job{" +
```

Comment:
Is this toString method simply for debugging and verifying the objects? If not this would be bad JSON formatting I believe.

Reply:
simply debugging

Reply:
When submitting a pull request do not include debugging code. Keep that for yourself in the future. By the way we could keep this toString method and make JSON toString() output a standard. This could make converting our application to a rest application without using additional JSON parsing libraries albeit we would only have one way parsing of from Object to JSON.